.github/release.yml

```
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - "main"
```

```suggestion
      - ["main","master"]
```
Not sure if Github does some magic for us here, but I _think_ that since we use the old main branch name, that we have to include "master" here.